packages/frontend-language-toggle/rollup.config.js

```diff
@@ -18,7 +18,9 @@ export default {
       targets: [
         { src: "./src/macro.njk", dest: "./build/" },
         { src: "./src/template.njk", dest: "./build/" },
+        { src: "./build/cjs/language-param-setter.d.ts", dest: "./build/cjs/", rename: "language-param-setter.d.cts" }
       ],
+      hook: "closeBundle"
```

Comment:
(Needs to be `closeBundle` so it runs after the rest of it has been output - which is the fudge)